frontend/src/components/products/CreateForm.jsx

```diff
@@ -0,0 +1,211 @@
+import React, { useState } from "react";
+import Card from "@material-ui/core/Card";
+import CardContent from "@material-ui/core/CardContent";
+import Button from "@material-ui/core/Button";
+import FormControl from "@material-ui/core/FormControl";
+import Select from "@material-ui/core/Select";
+import InputLabel from "@material-ui/core/InputLabel";
+import MenuItem from "@material-ui/core/MenuItem";
+import TextField from "@material-ui/core/TextField";
+import InputAdornment from "@material-ui/core/InputAdornment";
+import Input from "@material-ui/core/Input";
+import Chip from "@material-ui/core/Chip";
+import ImagesPreview from "./ImagesPreview";
+import { Grid, Container } from "@material-ui/core";
+import "./CreateForm.css";
+import { LOCATIONS, CATEGORIES } from "./constatnts";

```

Comment:
constatnts seems like a typo. Can you verify?

Reply:
Yes it was, nice catch. Fixed it